src/test/ui/destructuring-assignment/struct_destructure_fail.stderr

```diff
@@ -32,11 +32,11 @@ LL |     Struct { a, _ } = Struct { a: 1, b: 2 };
    |
 help: include the missing field in the pattern
    |
-LL |     Struct { a, b, _ } = Struct { a: 1, b: 2 };
+LL |     Struct { a, b _ } = Struct { a: 1, b: 2 };
    |               ^^^
```

Comment:
This is an unapplicable suggestion now :-/

Reply:
From @zackmdavis's description:

> The suggested code in the diff this commit introduces to destructuring-assignment/struct_destructure_fail.stderr doesn't work, but it didn't work beforehand, either (because of the "found reserved identifier _" thing), so you can't really call it a regression; it could be fixed in a separate PR.

Reply:
Ah! That's what I get for skip-reading.